router/wire.go

```diff
@@ -30,8 +30,9 @@ func Wire(deps Dependencies) http.Handler {
 	resultsController := results.NewController(deps.ResultsDataProvider)
 	r.GET("/v1/results", resultsController.Index)
 
-	problemsController := problems.NewController(deps.ProblemsDataProvider)
+	problemsController := problems.NewController(deps.ProblemsDataProvider, deps.UsersDataProvider)
 	r.GET("/v1/problems/sets", problemsController.SetIndex)
+	r.GET("/v1/users/:userId/problems/:problemId", problemsController.GetProblem)
```

Comment:
Can you change it here to be `userID` and `problemID`, so it's consistent with the rest of the code?

Reply:
Sure.